src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneCommands/SetAzureRmDataLakeStoreItemExpiry.cs

```diff
@@ -0,0 +1,92 @@
+
+using Microsoft.Azure.Commands.DataLakeStore.Models;
+using Microsoft.Azure.Commands.DataLakeStore.Properties;
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.DataLake.Store;
+
+namespace Microsoft.Azure.Commands.DataLakeStore
+{
+    [Cmdlet(VerbsCommon.Set, "AzureRmDataLakeStoreItemExpiry", SupportsShouldProcess = true, DefaultParameterSetName = BaseParameterSetName), OutputType(typeof(DataLakeStoreItem))]
+    [Alias("Set-AdlStoreItemExpiry")]
+    public class SetAzureRmDataLakeStoreItemExpiry : DataLakeStoreFileSystemCmdletBase
+    {
+        internal const string BaseParameterSetName = "Set expiry as Absolute or NeverExpire";
+        internal const string RelativeExpiryParameterSetName = "Set expiry as relative to creation or now";
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0,ParameterSetName = BaseParameterSetName, Mandatory = true,
+            HelpMessage = "The Data Lake Store account to perform the action in")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, ParameterSetName = RelativeExpiryParameterSetName, Mandatory = true,
+            HelpMessage = "The Data Lake Store account to perform the action in")]
+        [ValidateNotNullOrEmpty]
+        [Alias("AccountName")]
+        public string Account { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1,ParameterSetName = BaseParameterSetName, Mandatory = true,
+            HelpMessage = "The name of the firewall rule.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, ParameterSetName = RelativeExpiryParameterSetName, Mandatory = true,
+            HelpMessage = "The name of the firewall rule.")]
+        [ValidateNotNullOrEmpty]
+        public DataLakeStorePathInstance Path { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, ParameterSetName = BaseParameterSetName, Mandatory = false,
+            HelpMessage =
+                "The expiration time for the specified file. If no value or set to MaxValue, the file will never expire.")]
+        [ValidateNotNullOrEmpty]
+        public DateTimeOffset? Expiration { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2,ParameterSetName = RelativeExpiryParameterSetName, Mandatory = false,
+            HelpMessage = "Relative expiry options. RelativeToNow or RelativeToCreationDate are current options")]
+        [ValidateNotNullOrEmpty]
+        public DataLakeStoreEnums.PathRelativeExpiryOptions? RelativeFileExpiryOption { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 3, ParameterSetName = RelativeExpiryParameterSetName, Mandatory = false,
+            HelpMessage = "The relative time in milliseconds with respect to now or creation time")]
+        [ValidateNotNullOrEmpty]
+        public long? RelativeTime { get; set; }
```

Comment:
@rahuldutta90 nit: make sure there's an empty line between this parameter and `ExecuteCmdlet()`

Reply:
Fixed.